client/utils/icons.ts

```diff
@@ -0,0 +1,32 @@
+import { IconName } from 'components';
+
+export type PubPubIconName =
+	| 'overview'
+	| 'activity'
+	| 'page'
+	| 'layout'
+	| 'review'
+	| 'submission'
+	| 'connection'
+	| 'impact'
+	| 'member'
+	| 'settings'
+	| 'collection'
+	| 'pub'
+	| 'community';
+
+export const pubPubIcons: Record<PubPubIconName, IconName> = {
```

Comment:
This is my first crack at providing PubPub names for our mostly-blueprint set of icons. We should gradually introduce this mapping into our codebase so we don't always need to remember that e.g. the Submissions icon is `manually-entered-data`.